auth/http.go

```diff
@@ -433,19 +459,18 @@ func httpSendPermitsJson(central *ImqsCentral, users []authaus.AuthUser, ident2p
 		}
 		groups, err := authaus.DecodePermit(permit.Roles)
 		if err != nil {
-			authaus.HttpSendTxt(w, http.StatusInternalServerError, err.Error())
-			return
+			return nil, err
```

Comment:
There's the bit of me that believes that we should add some kind of error preamble in the style below
```suggestion
			return nil, fmt.Errorf("Could not decode permit: %v", err)
```

for the sake of debugging in future.